.eslintrc.json

```diff
@@ -28,6 +28,7 @@
     ],
     "rules": {
         "react/react-in-jsx-scope": "off",
-        "@typescript-eslint/no-inferrable-types": "off"
+        "@typescript-eslint/no-inferrable-types": "off",
+	"@typescript-eslint/no-empty-interface": "off"
```

Comment:
Do we need to disable this message for the entire project? Looks like you have it disabled just for the line below ... Can we bring it back?